lib/prometheus-metrics.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const client = require('prom-client');
+
+// The current tests has circuit names like:
+// 'circuit one' (with blank space) and others like
+// 3beb8f49-62c0-46e0-b458-dcd4a62d0f48.
+// So to avoid "Error: Invalid metric name" we are changing the
+// circuit name to pass the tests.
+// More details:
+// https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels
+function normalizePrefix (prefixName) {
+  prefixName = prefixName.replace(/ /g, '_');
+  prefixName = prefixName.replace(/-/g, '_');
```

Comment:
This can be written as a single statement instead of two. And instead of checking for a digit below, we can just always prefix with a string. So these two statements just become:

```suggestion
  return `circuit_${prefixName.replace(/[ |-]/g, '_')}_`;
```
And the following four lines can be removed.

Reply:
Yup. The idea was to put to work first, then improve later.